### ISSUE TITLE
sync fails and report wrong size

### ISSUE THREAD

**Issue by beer4duke** [beer4all/rclone#4]
this happens always for this file: `encoder/encoder_cases_test.go`???
It cannot make it through...

```
[root@ctadevjulien rclone]# (xrootd-devel) ./rclone sync ./lib atlastest:tmp/te
2020/05/27 22:55:40 ERROR : encoder/encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:40 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting files as there were IO errors
2020/05/27 22:55:40 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting directories as there were IO errors
2020/05/27 22:55:40 ERROR : Attempt 1/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:40 ERROR : encoder/encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:40 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting files as there were IO errors
2020/05/27 22:55:40 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting directories as there were IO errors
2020/05/27 22:55:40 ERROR : Attempt 2/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:40 ERROR : encoder/encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:41 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting files as there were IO errors
2020/05/27 22:55:41 ERROR : root://eosctaatlaspps:1094//eos/ctaatlaspps/tmp/te: not deleting directories as there were IO errors
2020/05/27 22:55:41 ERROR : Attempt 3/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 22:55:41 Failed to sync: corrupted on transfer: sizes differ 2048147 vs 4096
```
this is a checkout of commit 4f977c5e5bcd876af1cfbd5a2f4c1f60e193fe61

**Comment by beer4duke** [comment]
```
[root@ctadevjulien encoder]# (xrootd-devel) ../../../rclone/rclone sync encoder_cases_test.go atlastest:tmp/te
2020/05/27 23:09:24 ERROR : encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 ERROR : Attempt 1/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 ERROR : encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 ERROR : Attempt 2/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 ERROR : encoder_cases_test.go: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 ERROR : Attempt 3/3 failed with 1 errors and: corrupted on transfer: sizes differ 2048147 vs 4096
2020/05/27 23:09:24 Failed to sync: corrupted on transfer: sizes differ 2048147 vs 4096
```

This single file fails, why?

**Comment by beer4duke** [comment]
I did not manage to attach the file, but here is the md5sum:
```
[root@ctadevjulien encoder]# (xrootd-devel) md5sum encoder_cases_test.go
13eb50781dbff3f5e32983ae5a514284  encoder_cases_test.go
```

**Comment by beer4duke** [comment]
should be fixed with 30f3515f2dae1b0370171827bbdc4550ec088ccd
as the real underlying issue was the larger files copy
